examples/UKCore-Specimen-SampleState-Example.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Specimen xmlns="http://hl7.org/fhir">
+ <id value="UKCore-Specimen-SampleState-Example"/>
+ <status value="available"/>
+   <extension url="https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-SampleState">
+    <valueCodeableConcept>
+      <coding>
+        <system value="https://fhir.hl7.org.uk/CodeSystem/UKCore-SampleState" />
+        <code value="solid-tumour-sample" />
+        <display value="Solid Tumour Sample" />
+      </coding>
+  </extension>
+ <type>
+  <coding>
+   <system value="http://snomed.info/sct"/>
+   <code value="todo"/>
```

Comment:
?

Reply:
I'm hoping Omar will give a correct Specimen.type to go with the extension, i'm not sure if it should be something like "258435002 |Tumour tissue specimen" or "76261009 | Skin structure of axilla"